notify_slack/action.yml

```diff
@@ -8,63 +8,84 @@ inputs:
   slack_bot_token:
     description: Slack Bot Token
     required: true
+  include_commit_info:
+    description: Include commit information in slack notification
+    default: 'false'
+    required: false
 
 runs:
   using: composite
   steps:
-    - name: Set message
-      id: set_message
+    - name: Checkout
+      uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
```

Comment:
The long numbers always bothers me, is it some kind of SHA of commit?

Reply:
yes, it is one of the recommended best practices.
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions